### DISCUSSION TITLE
Fetching axes

### DISCUSSION THREAD

**Discussion by andrzejnovak** [scikit-hep/hist#178] [Ideas]
Is there a better way to fetch a specific axis than `[ax for ax in self.axes if ax.name == 'some_name']` if not, could `self.axes` return a named tuple instead?

**Comment by henryiii** [comment]
`self.axes['some_name']`?

**Reply by andrzejnovak** [reply]
Ah, it is a named tuple. I got confused by the repre, thanks.

**Reply by henryiii** [reply]
It's a custom tuple, not quite a named tuple. That's also why you can access the methods and properties "all-at-once", like h.axes.name will give a list of all names.
